CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ All notable changes to this project will be documented in this file.
 - Added a `check_surface_based_positions` positions method for verifying antenna
 positions are near surface of whatever celestial body their positions are referenced to
 (either the Earth or Moon, currently).
+- Added a switch to `UVData.write_ms` called `flip_conj`, which allows a user to write
+out data with the baseline conjugation scheme flipped from the standard `UVData`
+convention.
+- Added the `utils.determine_pol_order` method for determining polarization
+order based on a specified scheme ("AIPS" or "CASA").
```

Comment:
This should be at the top (reverse chronological).

Reply:
This is either a sign that I'm writing too many or too few PRs...